Userland/Libraries/LibGfx/Point.h

```diff
@@ -91,6 +91,13 @@ class Point {
         return point;
     }
 
+    [[nodiscard]] Point<T> scaled(T dboth) const
+    {
+        Point<T> point = *this;
+        point.scale_by(dboth);
+        return point;
+    }
```

Comment:
Hm, if I have a `Point<float>` and I call `::scaled<int>(3)`, I'd expect the scaling to happen first before converting the coordinates to a `Point<int>`. This can be a bit confusing?

Reply:
Can you explicitly called `scaled<int>(3)`? The `T` is the class template, so `Point<float>::scaled(3)` is only going to be `Point<float>::scaled<float>(3)`.

Reply:
Ah I misunderstood the commit message; this is the version of the API where you specify both dimensions at once with `dboth`. No type mixing going on here :^) carry on!